src/abstractdataframe/iteration.jl

```diff
@@ -99,13 +99,12 @@ struct DataFrameColumns{T<:AbstractDataFrame, V} <: AbstractVector{V}
 end
 
 """
-    eachcol(df::AbstractDataFrame, names::Bool=true)
+    eachcol(df::AbstractDataFrame, names::Bool=false)
 
 Return a `DataFrameColumns` that iterates an `AbstractDataFrame` column by column.
-If `names` is equal to `true` (currently the default, in the future the default
-will be set to `false`) iteration returns a pair consisting of column name
+If `names` is equal to `true` iteration returns a pair consisting of column name
 and column vector.
-If `names` is equal to `false` then column vectors are yielded.
+If `names` is equal to `false` (the default) then only column vectors are yielded.
```

Comment:
Put this sentence first (mentioning iteration in this one rather than in the one for `true`).

Reply:
fixed